### ISSUE TITLE
Accuracy = 0

### ISSUE THREAD

**Issue by robd2** [giannisnik/cnn-graph-classification#6]
Thank you so much @giannisnik for publishing this code. Is a very interesting use of a CNN. I'm trying to reproduce some of your results but keep getting Accuracy = 0 measurements. Any thoughts on where to focus my debugging? Maybe related to the missing "best model"?

![image](https://user-images.githubusercontent.com/11090772/45515262-a4476880-b775-11e8-9ccc-5dcb9c17c9c0.png)

**Comment by robd2** [comment]
just FYI, I think I've narrowed it down to the logic in the lambda function:
evaluation = lambda output, target: torch.sum(output.eq(target)) / target.size()[0]

seems to be returning an int rather than a float.

**Comment by Wangshuaiia** [comment]
thank you for your issue@robd2. I have  met this problem just now.
